test/e2e/fixtures/util.go

```diff
@@ -16,10 +17,9 @@ func runCli(diagnostics *Diagnostics, args []string) (string, error) {
 	runArgs := append([]string{"-n", Namespace}, args...)
 	cmd := exec.Command("../../dist/argo", runArgs...)
 	cmd.Env = os.Environ()
-	output, err := exec.Command("../../dist/argo", runArgs...).CombinedOutput()
-	stringOutput := string(output)
-	diagnostics.Log(log.Fields{"args": args, "output": stringOutput, "err": err}, "Run CLI")
-	return stringOutput, err
+	output, err := executil.RunCommandExt(cmd, executil.CmdOpts{Timeout: 30 * time.Second})
```

Comment:
rather than letting the test framework fail due to timeout, lets do it here instead, so only the test fails